.github/workflows/node.js.yml

```diff
@@ -25,5 +25,6 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm ci
-    - run: npm run build --if-present
+    - run: npm install -g gatsby-cli
+    - run: yarn install 
+    - run: npm run build
```

Comment:
even here you can use "yarn run build"

Reply:
indeed we should use the same tool for installing dependencies and actually building the site.